src/scripts/autoloads/global.gd

```diff
@@ -84,6 +84,8 @@ var is_connected_to_resotocore:bool	= false
 var content_manager: ContentManager	= null
 var popup_manager: PopupManager		= null
 
+var authorized : bool = true
```

Comment:
sounds like the wrong default.

Reply:
It was easier to have a true default and let it change to false if the request was unauthorized, than having it as a false default, and set it to true right away when we connect. But ill change it.